product.json

```diff
@@ -45,6 +45,7 @@
 		"Microsoft.cms",
 		"Microsoft.dacpac",
 		"Microsoft.import",
+		"microsoft.managed-instance-dashboard",
```

Comment:
casing of Microsoft is different than all the other extensions.  I'd be consistent if there's no reason not to be.

Reply:
fixed